### ISSUE TITLE
Add all the new sponsor categories

### ISSUE THREAD

**Issue by Galactic-Dev** [Galactic-Dev/iSponsorBlock#49]
Add all the sponsor categories from https://wiki.sponsor.ajay.app/w/Types that are currently missing in the latest version of iSponsorBlock.

This request was originally brought up in issues #31 and #33.

**Comment by asportnoy** [comment]
Is this being worked on? Would love to have the new categories.

**Comment by Galactic-Dev** [comment]
> Is this being worked on? Would love to have the new categories.

Yes I am currently working on it. I've implemented most of the new functionality but there are a few things I'm still working on.

**Comment by asportnoy** [comment]
Hey, any updates?

**Comment by dayanch96** [comment]
@asportnoy @yodaluca23 can I have link to video with highlight to test it?

**Comment by asportnoy** [comment]
https://www.youtube.com/watch?v=sSiuW1HcGjA

**Comment by yodaluca23** [comment]
> @asportnoy @yodaluca23 can I have link to video with highlight to test it?

https://youtu.be/IL2P1IB-2nc

**Comment by dayanch96** [comment]
Thanks. I already added Filler and previews. Will work on highlights and push it all together in one pr

**Comment by dayanch96** [comment]
Almost done

![E560338E-1B7B-43FE-AE0D-C3D90AA8BEF6](https://github.com/Galactic-Dev/iSponsorBlock/assets/38832025/84e6cd52-0b5f-4901-8851-5e13dc2db872)

**Comment by dayanch96** [comment]
@asportnoy @yodaluca23 sorry to be late with the reply. I uploaded these categories in my fork but it is beta - highlights doesn't work if there activated other categories before highlight. The fork will be sent to this repository as pr when I find time and desire to fix the problem with the highlights

**Comment by Jecobe** [comment]
is this gonna still happen

**Comment by olivertzeng** [comment]
yeah is this

> is this gonna still happen

**Comment by PoomSmart** [comment]
Added `preview`.

**Comment by PoomSmart** [comment]
Added hook, filler and highlight in the latest commit.
